docs/source/conf.py

```diff
@@ -106,20 +106,29 @@
 html_theme = "furo"
 html_title = "Flyte"
 
+announcement = """
+📢 This is the old documentation for Flyte.
```

Comment:
why do we change it here?

Reply:
@pingsutw I didn't do it manually. I'm confused why this happened too. My local repository's conf file doesn't have these changes.

Reply:
are you able to remove them manually?